develop: add proxysql integration (#10022)
pingcap/docs: develop/dev-guide-proxysql-integration.md

# Integrate TiDB with ProxySQL

<CustomContent platform="tidb">

This document describes how to integrate **TiDB** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. For more information, refer to:

- [TiDB Documentation](/overview.md)

</CustomContent>

<CustomContent platform="tidb-cloud">

> This document describes how to integrate **TiDB Cloud** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. For more information, refer to:
> 
> - [TiDB Cloud Documentation](/tidb-cloud/tidb-cloud-intro.md)

Reviewer: ```suggestion
This document describes how to integrate **TiDB Cloud** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. 
```

Reviewer: When users are viewing this doc, they are already opening the Cloud docs website, so there is no need to direct them to the Cloud introduction again.

</SimpleTab>

</CustomContent>

<CustomContent platform="tidb-cloud">

> It is recommended to use [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) directly when you need hosting TiDB services (for example, you cannot manage it yourself, or you need a cloud-native environment). To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).

Reviewer: ```suggestion
To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).
```

</SimpleTab>

</CustomContent>

<CustomContent platform="tidb-cloud">

> To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).

Reviewer: ```suggestion
To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).
```
